### PR TITLE
deepcopygen for v1alpha4

### PR DESCRIPTION
**Pull request by BenTheElder**
xref: #1083 
I missed this when adding the v1alpha4 types

### CODE REVIEW

**Comment by BenTheElder** [comment]
FYI @robertojrojas, breaking this off as a small unrelated fix

File: hack/update/generated.sh [hunk 1]
```diff
@@ -46,6 +46,8 @@ cd "${FAKE_REPOPATH}"
 # run the generators
 bin/deepcopy-gen -i ./pkg/internal/apis/config/ -O zz_generated.deepcopy --go-header-file hack/tools/boilerplate.go.txt
 bin/deepcopy-gen -i ./pkg/apis/config/v1alpha3 -O zz_generated.deepcopy --go-header-file hack/tools/boilerplate.go.txt
+bin/deepcopy-gen -i ./pkg/apis/config/v1alpha4 -O zz_generated.deepcopy --go-header-file hack/tools/boilerplate.go.txt
```
Comment (aojea) [inline hack/update/generated.sh]: :upside_down_face:

**Pull request merged**